packages/near-membrane-base/src/__tests__/environment.spec.ts

```diff
@@ -0,0 +1,149 @@
+import { VirtualEnvironment } from '../environment';
+
+describe('VirtualEnvironment', () => {
+    afterEach(() => {
+        jest.resetAllMocks();
+    });
+
+    it('throws when options bag is missing', () => {
+        expect.assertions(1);
+        expect(() => {
+            // @ts-ignore
+            // eslint-disable-next-line no-new
+            new VirtualEnvironment();
+        }).toThrow();
```

Comment:
Do we care about what type of error is thrown?

Reply:
The type is just `Error`, but we could assert something about the message, ie. `}).toThrow(/Missing VirtualEnvironmentOptions options bag/);`

Reply:
Fixed!

Reply:
Ah ok. Ya that it contains the VirtualEnvironmentOption keyword would narrow it down without getting too caught up on wording (which changes from time to time).